example/3-the_final_chapter.md

```diff
@@ -5,4 +5,4 @@
 | 1    |   2 |
 | john | doe |
 
-![Some figure](./image.png)
+![Some figure](image.png)
```

Comment:
Maybe we should actually add an image here

Reply:
Done.